CMakeLists.txt

```
@@ -1,33 +1,27 @@
-#-----------------------------------------------------------------------------
-set(MODULE_NAME SlicerMONAIViz)
-
-#-----------------------------------------------------------------------------
-set(MODULE_PYTHON_SCRIPTS
-  ${MODULE_NAME}.py
-  ${MODULE_NAME}Lib/__init__.py
-  ${MODULE_NAME}Lib/utils.py
-  )
-
-set(MODULE_PYTHON_RESOURCES
-  Resources/Icons/${MODULE_NAME}.png
-  Resources/UI/${MODULE_NAME}.ui
-  )
-
-#-----------------------------------------------------------------------------
-slicerMacroBuildScriptedModule(
-  NAME ${MODULE_NAME}
-  SCRIPTS ${MODULE_PYTHON_SCRIPTS}
-  RESOURCES ${MODULE_PYTHON_RESOURCES}
-  WITH_GENERIC_TESTS
-  )
-
-#-----------------------------------------------------------------------------
-if(BUILD_TESTING)
-
-  # Register the unittest subclass in the main script as a ctest.
-  # Note that the test will also be available at runtime.
-  slicer_add_python_unittest(SCRIPT ${MODULE_NAME}.py)
-
-  # Additional build-time testing
-  add_subdirectory(Testing)
-endif()
+cmake_minimum_required(VERSION 3.16.3...3.19.7 FATAL_ERROR)
+
+project(MONAIViz)
+
+#-----------------------------------------------------------------------------
+# Extension meta-information
+set(EXTENSION_HOMEPAGE "https://github.com/Project-MONAI/SlicerMONAIViz#slicermonaiviz")
+set(EXTENSION_CATEGORY "Active Learning")
+set(EXTENSION_CONTRIBUTORS "NVIDIA")

```

this is not an Nvidia product, how about:
```suggestion
set(EXTENSION_CATEGORY "Developer Tools")
set(EXTENSION_CONTRIBUTORS "MONAI Consortium")
```

cc @SachidanandAlle

---

I copied metadata from the module. I would suggest to list individual contributors as in all other Slicer modules. Once you agreed on this, also update this information for the module in MONAIViz.py.